loaders.py

```diff
@@ -9,6 +9,8 @@
 from attrdict import AttrDict
 from skimage.transform import resize
 from torch.utils.data import Dataset, DataLoader
+from sklearn.externals import joblib
+from cv2 import resize as resize_cv
```

Comment:
@apyskir why exactly do we want 2 resize flavours?

Reply:
`skimage.transform.resize` accepts only numbers [-1,1]. Distances don't match this requirement. Maybe I could divide the distance matrix `Di` by `Di.max()` and after resize multiply by this number, but I am not sure if it is OK. I prefer as little resize/rescale operations on `Di` as possible